__tests__/actions/URL.spec.ts

```diff
@@ -31,4 +42,20 @@ describe('URL function', () => {
     expect(actual).toEqual(expected);
   });
 
+  it('inverts a URL action', () => {
+    if (URL.invert) {
+      const action = {
+        WFWorkflowActionIdentifier: 'is.workflow.actions.url' as WFWorkflowActionIdentifier,
+        WFWorkflowActionParameters: {
+          WFURLActionURL: '' as WFWorkflowActionParameters['WFURLActionURL'],
```

Comment:
This line doesn't need the `as WFWorkflowActionParameters['WFURLActionURL']`. There's an argument for keeping it, but I'd prefer to get rid of it for now as `WFURLActionURL` is just a string anyway.

This means that this line will simply become:
```js
WFURLActionURL: '',
```

You'll then be able to remove the `WFWorkflowActionParameters` import on line 4 as well then.